tools/PostProcessing/read_module.f90

```diff
@@ -458,10 +458,10 @@ subroutine read_eigenvalues
     efermi = zero
     if(nspin==1) then
        read(17,fmt='(a6,f18.10)') str,efermi(1)
-       write(*,fmt='(4x,"Fermi level: ",f12.5," Ha")') efermi(1)
+       write(*,fmt='(4x,"Fermi level: ",f12.5," Ha   (=",f7.3," eV)")') efermi(1), efermi(1)*HaToeV
```

Comment:
I wonder about the formatting here: we might need more digits before the decimal point when using eV than using Ha.  Should we change it to `f10.3` maybe?